### PR TITLE
DDCE-198: Play 2.6 Upgrade

### PR DESCRIPTION
**Pull request by Ufuoma-Daniel**
# DDCE-198

##Bug fix / New feature (delete as appropriate)
Acceptance test changes required and should be merged first - https://github.com/hmrc/ras-acceptance-tests/pull/75

## PR Suggestions
- Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?
- Have you done a visual check of the changes?
- Is there any still commented or unused code? Lingering printlns?
- Have things been implemented in a complicated way?
- Are the test still over the coverage threshold?
- Does the compiler throw a lot of warnings? 
- Have methods and tests been named clearly?
- Were there any changes in the config? Do changes need to be made in app-config-???

## Checklist PR Raiser
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date
 
## Checklist PR Reviewer
 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: app/controllers/FileUploadController.scala [hunk 1]
```diff
@@ -223,33 +225,35 @@ class FileUploadController @Inject()(fileUploadConnector: FileUploadConnector,
...
 
   private def extractErrorReason(uploadResponse: Option[UploadResponse]):String ={
-    uploadResponse match {
+		val fileUploadEmptyFileReason= "Envelope does not allow zero length files, and submitted file has length 0"
+
+		uploadResponse match {
       case Some(response) =>
         response.code match {
-          case "400" if response.reason.getOrElse("").contains(Messages("file.upload.empty.file.reason")) =>
+          case "400" if response.reason.getOrElse("").contains(fileUploadEmptyFileReason) =>
```
Comment (fraser-sprott) [inline app/controllers/FileUploadController.scala:233]: any reason this is no longer in the messages?

File: app/controllers/RasResidencyCheckerController.scala [hunk 2]
```diff
@@ -90,10 +93,10 @@ trait RasResidencyCheckerController extends RasController with AuditService {
 
   private[controllers] def extractResidencyStatus(residencyStatus: String): String = {
     (residencyStatus, apiVersion) match {
-      case (SCOTTISH, _) => Messages("scottish.taxpayer")
-      case (WELSH, ApiV2_0) => Messages("welsh.taxpayer")
-      case (OTHER_UK, ApiV1_0) => Messages("non.scottish.taxpayer")
-      case (OTHER_UK, ApiV2_0) => Messages("non.scottish.taxpayer")
+      case (SCOTTISH, _) => "Scotland"
```
Comment (fraser-sprott) [inline app/controllers/RasResidencyCheckerController.scala:96]: Same as before, any reason no longer in messages?

File: app/controllers/FileUploadController.scala (hunk 1 above)
Comment (Ufuoma-Daniel) [reply app/controllers/FileUploadController.scala:233]: I've had issues with keys being converted to messages well before they need to e.g. having welsh being fed into a function. I tend to remove Messages to the view and have keys be passed around until that point.

Comment (fraser-sprott) [reply app/controllers/FileUploadController.scala:233]: yeah, i'd agree but the fileUploadEmptyFileReason is a lengthy string, looks like something from a message file

**Pull request merged**